packages/compiler/test/checker/augment-decorators.test.ts

```diff
@@ -170,6 +171,74 @@ describe("compiler: checker: augment decorators", () => {
     it("interface", () => expectTarget(`@test("target") interface Foo { }`, "Foo"));
     it("operation in interface", () =>
       expectTarget(`interface Foo { @test("target") list(): void }`, "Foo.list"));
+    it("uninstantiated template", async () => {
```

Comment:
Can you add a test to make sure Foo<string>.bar doesn't work either (might need to make Foo<string> an alias.

Reply:
I added the alias test.